models/intel/machine-translation-nar-en-ru-0002/README.md

```diff
@@ -0,0 +1,42 @@
+# machine-translation-nar-en-ru-0002
+
+## Use Case and High-Level Description
+
+This is a English-Russian machine translation model based on non-autoregressive Transformer topology.
+
+Tokenization occurs using the SentencePieceBPETokenizer (see the demo code for implementation details) and the enclosed tokenizer_src and tokenizer_tgt folders.
+
+## Specification
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| GOps              | 23.17                 |
+| MParams           | 69.29                 |
+| Source framework  | PyTorch\*             |
+
+## Accuracy
+
+The quality metrics were calculated on the wmt19-ru-en dataset ("test" split in lower case).
+
+| Metric                    | Value         |
+|---------------------------|---------------|
+| BLEU                      |        21.6 % |
```

Comment:
update accuracy value in README files and reference in AC configs, please

Reply:
Done